toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -329,8 +329,8 @@ func (r *RpmRepoCloner) WhatProvides(pkgVer *pkgjson.PackageVer) (packageName st
 				continue
 			}
 			// Local sources are listed last, keep searching for the last possible match
-			packageName = matches[1]
-			logger.Log.Debugf("'%s' is available from package '%s'", pkgVer.Name, packageName)
+			packageNames = append(packageNames, matches[1])
+			logger.Log.Debugf("'%s' is available from package '%s'", pkgVer.Name, packageNames)
```

Comment:
Should this be a `%v` as `packageNames` is a slice:
```suggestion
			logger.Log.Debugf("'%s' is available from package '%v'", pkgVer.Name, packageNames)
```

Reply:
Another good catch!